WeatherPredictorApi/SQLApp/crud.py

```diff
@@ -0,0 +1,74 @@
+from xmlrpc.client import Boolean
```

Comment:
This file is generally called dbContext. The context is the file where you manipulate and work with queries